packages/web/src/pages/links/LinksPage.tsx

```diff
@@ -28,7 +28,3 @@ export const LinksPage = () => {
     </div>
   );
 };
-
-// we export default pages for code splitting later
-// more info at https://reactjs.org/docs/code-splitting.html
-export default LinksPage;
```

Comment:
Actually, I decided to add `export default LinksPage;` to `index.tsx`. Not much difference here.